apps/condo/domains/contact/components/ContactsEditor/index.tsx

```diff
@@ -57,6 +59,21 @@ export interface IContactEditorProps {
     property?: string,
     unitName?: string,
     allowLandLine?: boolean;
+    disabled?: boolean
+}
+
+const ContactsInfoFocusContainer = styled(FocusContainer)`
+  position: relative;
+  left: ${({ padding }) => padding ? padding : '24px'};
+  box-sizing: border-box;
+  width: 100%;
+  background: ${colors.backgroundLightGrey};
+`
+const { TabPane } = Tabs
+
+enum CONTACT_EDITOR_TABS {
+    FROM_RESIDENT = '0',
```

Comment:
You don't need to specify value of enum because it starts with 0 by default. Or you want to represent value as string may be?

Reply:
the value of the `defaultActiveKey` prop is a string